<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CampaignEditorClass</class>
 <widget class="QMainWindow" name="CampaignEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1206</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CampaignEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>1091</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="widget_mainPage">
     <widget class="QLabel" name="label_mainPage_title">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>851</width>
        <height>241</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>36</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Campaign Editor</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>210</y>
        <width>441</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layout_mainMenu">
       <item>
        <widget class="QPushButton" name="button_mainPage_newCampaign">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>New Campaign</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_mainPage_loadCampaign">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Load Campaign</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_mainPage_exit">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_createNewCampaign">
     <widget class="QLabel" name="label_createNewCampaign_title">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>701</width>
        <height>241</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>36</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>New Campaign</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="newCampaignInput">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>571</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="input_createNewCampaign_campaignName">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>401</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_createNewCampaign_nameInput">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Campaign Name</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_createNewCampaign_createCampaign">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>100</y>
         <width>221</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_addMap">
     <widget class="QPushButton" name="button_addMap_load">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>390</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Goudy Stout</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add Map</string>
      </property>
     </widget>
     <widget class="QListView" name="list_addMap_mapList">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>361</width>
        <height>511</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Goudy Stout</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_addMap_title">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add Map Menu</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="input_addMap_mapNumber">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>230</y>
        <width>111</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Goudy Stout</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="button_addMap_exit">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>470</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Goudy Stout</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_loadCampaign">
     <widget class="QPushButton" name="button_loadCampaign_load">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>400</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>load Campaign</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_loadCampaign_exit">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>480</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_loadCampaign_title">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Load Campaign Menu</string>
      </property>
     </widget>
     <widget class="QListView" name="list_loadCampaign_campaignList">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>361</width>
        <height>511</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Goudy Stout</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_campaignEditor">
     <widget class="QLabel" name="label_campaignEditor_title">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Campaign Editor</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_campaignEditor_save">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>520</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_campaignEditor_exit">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>520</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="box_campaignEditor_changeOrder">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>90</y>
        <width>441</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Change order</string>
      </property>
      <widget class="QLabel" name="label_campaignEditor_firstMap">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>1st Map</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="input_campaignEditor_secondMap">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>90</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="input_campaignEditor_firstMap">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_campaignEditor_secondMap">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>2nd Map</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_campaignEditor_changeOrder">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>90</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Place Wall</string>
       </property>
      </widget>
     </widget>
     <widget class="QListView" name="list_campaignEditor_mapList">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>341</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>320</y>
        <width>441</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Remove map</string>
      </property>
      <widget class="QPushButton" name="button_campaignEditor_removeMap">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>70</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="input_campaignEditor_mapNumber">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_campaignEditor_mapNumber">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Map Number</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="button_campaignEditor_addMap">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>150</y>
        <width>75</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Map</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1206</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
